modules/base-libraries/pages/Text.adoc

```diff
@@ -51,7 +51,25 @@ Returns the text value containing the sequence of characters in `cs`.
 func size(t : Text) : Nat
 ----
 
-Returns `t.size()`, the number of characters in `t` (and `t.chars()`).
+[[value.toLower]]
```

Comment:
Did you intentionally delete the line about `size()`?

Reply:
oops sorry, fat fingers :). Nope not intentional